swagger-to-har.js

```diff
@@ -29,13 +29,26 @@ var Instantiator = require('./schema-instantiator.js')
  * @return {Object}         HAR Request object
  */
 var createHar = function (swagger, path, method) {
+  return createHarWithQueryValues(swagger, path, method, {})
+}
+
+/**
+ * Create HAR Request object for path and method pair described in given swagger.
+ *
+ * @param  {Object} swagger           Swagger document
+ * @param  {string} path              Key of the path
+ * @param  {string} method            Key of the method
+ * @param  {Object} queryParamValues  Values for the query parameters if present
+ * @return {Object}                   HAR Request object
+ */
+var createHarWithQueryValues = function (swagger, path, method, queryParamValues) {
```

Comment:
Here, you might just check `typeof queryParamValues === 'object'` to see of they were provided and act correspondingly.